assets/lib/share-link/share-link.js

```diff
@@ -58,32 +56,24 @@
 			} );
 		};
 
-		var initSettings = function() {
-			$.extend( settings, ShareLink.defaultSettings, userSettings );
+		const initSettings = function() {
+			settings = { ...ShareLink.defaultSettings, ...userSettings };
 
-			[ 'title', 'text' ].forEach( function( propertyName ) {
-				settings[ propertyName ] = settings[ propertyName ].replace( '#', '' );
-			} );
+			[ 'title', 'text' ].forEach( ( propertyName ) => settings[ propertyName ] = settings[ propertyName ].replace( '#', '' ) );
 
 			settings.classPrefixLength = settings.classPrefix.length;
 		};
 
-		var initElements = function() {
-			$element = $( element );
-		};
-
-		var init = function() {
+		const init = function() {
```

Comment:
Maybe convert to arrow function?

Reply:
@hein-obox it can be an arrow function, but for the sake of style consistency throughout the ShareLink object, I kept it as a regular function